up-l1/zenoh.adoc

```diff
@@ -98,22 +108,24 @@ Different message types **MUST** use different Zenoh API.
 
 === URI Mapping
 
-UUri **MUST** be able to be serialized into link:../basics/uri.adoc#42-micro-uris[micro form].
-Then we can transform the micro-form UUri, which is `u8` array, into string in lowercase hexadecimal representation.
+UUri **MUST** be able to be serialized into link:../basics/uri.adoc#short-uris[short form].
+Then we can transform the short-form UUri into string for Zenoh key.
 
 Zenoh key for uProtocol consists of three parts and they are separated by `/`.
-Zenoh key **MUST** start with `upl/` or `upr/`.
-While the local UUri maps to `upl/`, the remote UUri maps to `upr/`.
-If UAuthority is defined in UUri, it **MUST** be put in the second part.
-Then the rest of the UUri **MUST** be put in the last part.
 
-To be more specific:
+* `up/`: Zenoh key **MUST** always start with `up/` and the scheme (`up://`) in the short UUri is always removed.
```

Comment:
Serialize to short and long with Java doesn't add the scheme, does up-rust add the scheme? Perhaps we should remove it as it was optional anyways.

Reply:
You're right. No scheme while serializing to short and long in up-rust. I'll remove this description.